tests/integration/docker-compose.integration.yml

```
@@ -0,0 +1,36 @@
+services:
+  server:
+    build:
+      context: ../..
+      dockerfile: Dockerfile.server
+    environment:
+      - STREAMDL_GRPC_PORT=50051
+      - LOG_LEVEL=DEBUG
+      - PUID=1000
+      - PGID=1000
+    healthcheck:
+      test: ["CMD", "curl", "--fail", "--max-time", "5", "http://localhost:8080/health"]
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)